uniffi_bindgen/Cargo.toml

```diff
@@ -18,7 +18,7 @@ ffi-trace = ["uniffi_testing?/ffi-trace"]
 
 [dependencies]
 anyhow = "1"
-askama = { version = "0.12", default-features = false, features = ["config"] }
+rinja = { version = "0.3", default-features = false, features = ["config"] }
```

Comment:
I think this actually requires 0.3.4. With older versions I run into

```
   --> uniffi_bindgen-0.29.0/src/bindings/python/gen_python/mod.rs:186:10
    |
186 | #[derive(Template)]
    |          ^^^^^^^^
    |          |
    |          expected `&String`, found `String`
    |          expected due to the type of this binding
    |
    = note: this error originates in the derive macro `Template` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
Can you file this as a bug?